src/dialects/mssql/index.js

```diff
@@ -61,25 +61,32 @@ assign(Client_MSSQL.prototype, {
 
     // TODO: remove mssql implementation all together and use tedious directly
 
+    /* istanbul ignore next */
     const mssqlVersion = require('mssql/package.json').version;
-    if (mssqlVersion !== '4.1.0') {
-      throw new Error(
-        'This knex version does not support any other mssql version except 4.1.0 ' +
-          '(knex patches bug in its implementation)'
-      );
+    /* istanbul ignore next */
+    if (mssqlVersion === '4.1.0') {
+      mssqlTedious.ConnectionPool.prototype.release = release;
+      mssqlTedious.ConnectionPool.prototype._poolCreate = _poolCreate;
+    } else {
+      const [major, minor, patch] = mssqlVersion.split('.');
+      // if version is not ^4.2.2
+      if (major != 4 || minor < 2 || (minor == 2 && patch < 2)) {
```

Comment:
this logic should be updated to expect >= 5.0.0

Reply:
Done